EasyRestProjectNetTeam2.sln

```diff
@@ -1,11 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.2.32616.157
+# Visual Studio Version 16
```

Comment:
It is not recommended to push to the remote repository changes like that because other team members have another VS version, so every PR could have such differences.

Reply:
Please leave it as is, no need to make changes in the PR but be careful in the future